packages/jest-worker/src/workers/NodeThreadsWorker.ts

```diff
@@ -42,6 +45,8 @@ export default class ExperimentalWorker implements WorkerInterface {
   private _resolveExitPromise!: () => void;
   private _forceExited: boolean;
 
+  private _heartbeatTimeout: any;
```

Comment:
comments from child apply in this file as well (`any`, `timeout` vs `interval` etc)